src/EditorFeatures/VisualBasicTest/Wrapping/ChainedExpressionWrappingTests.vb

```diff
@@ -0,0 +1,258 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.Wrapping
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Wrapping
+    Public Class ChainedExpressionWrappingTests
+        Inherits AbstractWrappingTests
+
+        Protected Overrides Function CreateCodeRefactoringProvider(workspace As Workspace, parameters As TestParameters) As CodeRefactoringProvider
+            Return New VisualBasicWrappingCodeRefactoringProvider()
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)>
+        Public Async Function TestMissingWithSyntaxError() As Task
+            Await TestMissingAsync(
+"class C
+    sub Bar()
+        [||]the.quick().brown.fox(
+    end sub
+end class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)>
+        Public Async Function TestMissingWithoutEnoughChunks() As Task
+            Await TestMissingAsync(
+"class C
+    sub Bar()
+        [||]the.quick()
+    end sub
+end class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)>
+        Public Async Function TestWithEnoughChunks() As Task
+            Await TestAllWrappingCasesAsync(
```

Comment:
💡 Consider adding cases for wrapping without alignment

Reply:
done. #resolved.